packages/teams-js/src/private/externalAppCardActionsForCEA.ts

```diff
@@ -0,0 +1,106 @@
+   * @param appId ID of the application the request is intended for. This must be a UUID
+   * @param conversationId To tell the bot what conversation the calls are coming from
+   * @param url The URL to open
+   * @throws Error if the response has not successfully completed
+   * @returns Promise that resolves to ActionOpenUrlType indicating the type of URL that was opened on success and rejects with ActionOpenUrlError if the request fails
+   */
+  export async function processActionOpenUrl(
+    appId: AppId,
+    conversationId: string,
+    url: URL,
+  ): Promise<ActionOpenUrlType> {
+    ensureInitialized(runtime, FrameContexts.content);
+    if (!isSupported()) {
+      throw errorNotSupportedOnPlatform;
+    }
+    validateId(conversationId, new Error('conversation id is not valid.'));
+    const [error, response] = await sendMessageToParentAsync<[ActionOpenUrlError, ActionOpenUrlType]>(
+      getApiVersionTag(
+        externalAppCardActionsTelemetryVersionNumber,
+        ApiName.ExternalAppCardActionsForCEA_ProcessActionOpenUrl,
+      ),
+      ApiName.ExternalAppCardActionsForCEA_ProcessActionOpenUrl,
+      [appId, url.href, conversationId],
+    );
+    if (error) {
+      throw error;
+    } else {
+      return response;
+    }
+  }
+
+  /**
+   * @beta
+   * @hidden
+   * Delegates an Adaptive Card Action.Submit request to the host for the application with the provided app ID
+   * @internal
+   * Limited to Microsoft-internal use
+   * @param appId ID of the application the request is intended for. This must be a UUID
+   * @param conversationId To tell the bot what conversation the calls are coming from
+   * @param actionSubmitPayload The Adaptive Card Action.Submit payload
+   * @throws Error if host notifies of a error
+   * @returns Promise that resolves when the request is completed and rejects with ActionSubmitError if the request fails
+   */
+  export async function processActionSubmit(
+    appId: AppId,
+    conversationId: string,
+    actionSubmitPayload: IAdaptiveCardActionSubmit,
+  ): Promise<void> {
+    ensureInitialized(runtime, FrameContexts.content);
+    if (!isSupported()) {
+      throw errorNotSupportedOnPlatform;
+    }
+    validateId(conversationId, new Error('conversation id is not valid.'));
+    const error = await sendAndUnwrap<ActionSubmitError | undefined>(
+      getApiVersionTag(
+        externalAppCardActionsTelemetryVersionNumber,
+        ApiName.ExternalAppCardActionsForCEA_ProcessActionSubmit,
+      ),
+      ApiName.ExternalAppCardActionsForCEA_ProcessActionSubmit,
+      [appId, conversationId, actionSubmitPayload],
+    );
+    if (error) {
+      throw error;
+    }
+  }
+
+  /**
+   * @hidden
+   * Checks if the externalAppCardActionsForCEA capability is supported by the host
+   * @returns boolean to represent whether externalAppCardActions capability is supported
+   *
+   * @throws Error if {@linkcode app.initialize} has not successfully completed
+   *
+   * @internal
```

Comment:
This function is also missing the @beta tag, please make sure to add the same tags to every exported item

Reply:
updated the @beta tag with all the data models and functions.